src/OrbitQt/orbitmainwindow.cpp

```diff
@@ -1059,14 +1061,24 @@ void OrbitMainWindow::on_actionOpen_Capture_triggered() {
     return;
   }
 
-  OpenCapture(file.toStdString());
+  const bool starts_with_file_target = (target_configuration_.index() == 2);
+  const bool has_data = app_->HasCaptureData();
+  // If the current OrbitMainWindow starts with a FileTarget and the capture file is cleared, open
+  // new capture file in the current OrbitMainWindow. Otherwise, open capture files in new windows.
+  OpenCapture(file.toStdString(), !starts_with_file_target || has_data);
```

Comment:
Do you think the check for an empty capture could be confusing to the user? I get the intention, but as a user, I may wonder why a capture was suddenly opened in the existing window instead of a new one. Did you consider any alternatives to this?

Reply:
I'm considering the case that the user start an Orbit instance with "Loading capture" and then he "Clear capture". In such a case, the current `OrbitMainWindow` becomes useless (unless he ends the current session): the user cannot take a new capture, and; "Open capture" will open the selected capture file in a new window. 
I'm not sure whether it is a good idea to treat this case differently. So I put this part in a separate commit.

May be still open capture file in new window, and meanwhile provide a hint to tell the user he needs to end this session to reuse this window?

Reply:
Right, the rationale that the current window basically becomes useless makes sense, I agree. I wonder if, in case there is still a capture loaded, we want to ask the user if they want to open in a new window or re-use the current one.

We could also think about disabling the "Clear Capture" functionality when a capture is loaded from file. Or more generally, do we even need "Clear Capture" at all? With the current UI and this new window functionality, I can't really see a use-case for it. WDYT?

Reply:
I agree to disable the "Clear Capture" functionality completely. If a stadia instance is connected, the user can trigger a new capture to clear the current one. And the user can always "End Session" to reload a new capture file. 

Maybe we should also make the "End Session" button more obvious.

Reply:
Just realize that another case would be the user fails to load a capture due to the wrong format, and the current window also becomes useless. Fixing http://b/178071836 may help to address the problem.

Reply:
Sounds good. I think you should bring that up in the Standup today so see if anyone objects to remove the "Clear Capture" button. We could maybe still show it in devmode.

Reply:
Created http://b/179470701 for removing the "Clear Capture" button.